Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/MaterialComponentController.cpp

```diff
@@ -568,6 +583,13 @@ namespace AZ
                 // If the asset ID is different than what's already assigned then replace it
                 if (materialIt->second.m_materialAsset.GetId() != materialAssetId)
                 {
+                    // TODO: Although this works, we need to make sure that the previous material AssetId
+                    // is not being used in other slots before Disconnecting from the AssetBus.
+                    //const auto& prevAssetId = materialIt->second.m_materialAsset.GetId();
+                    //if (prevAssetId.IsValid())
+                    //{
+                    //    AZ::Data::AssetBus::MultiHandler::BusDisconnect(prevAssetId);
+                    //}
```

Comment:
Is there a ticket for this or is your intention to follow up immediatel?

Reply:
I'll follow up immediately, but need to consult with @gadams3.